    <html>
<head>
<title>Endosymbiont genomes</title>
</head>

<body bgcolor="#F8F8F8" background="image001.jpg" lang=EN-US
link=blue vlink=purple style='tab-interval:.5in'>

<div class=Section1>

<p align=center><b><span style='font-size:24.0pt;font-family:TimesNewRomanPS-BoldMT;
mso-bidi-font-family:TimesNewRomanPS-BoldMT'>
Genomes of N<sub>2</sub>-fixing endosymbionts and host-independence
</span></b></p>

<p align=center>
<span lang=PT style='font-size:14.0pt;font-family:
TimesNewRomanPSMT;mso-bidi-font-family:TimesNewRomanPSMT;mso-ansi-language:
PT'>
<a href="mailto:ElhaiJ@VCU.Edu">Jeff Elhai</a>
</p>

<p style='text-align:center'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;mso-bidi-font-family:
TimesNewRomanPSMT'>
(Submitted to BMC Genomics)</span>
</p>

<p style='margin-top:0in;margin-right:152.45pt;margin-bottom:
0in;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<a href="N-fixing-endosymbionts-Abstract.pdf">Abstract</a>
</span>
</p>

<p style='margin-right:152.45pt;margin-bottom:
0in;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'><span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<a href="N-fixing-endosymbionts-250819.pdf">Article</A> (including embedded tables and figures and references)
</span></p>

<p style='margin-top:0in;margin-right:152.45pt;margin-bottom:
.0001pt;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<a href="N-fixing-endosymbionts-references.pdf">References</a>
</span>
</p>

<!--
<p style='margin-right:152.45pt;margin-bottom:
0in;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<U>Figures and Tables</U></span></p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-1.pdf">Fig. 1</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Model and proposed realization of a reaction-diffusion system</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-2.pdf">Fig. 2</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Overview of phylogenetic tree, rooted by Gloeobacter violaceus PCC 7421</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-3.pdf">Fig. 3</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Phylogenetic tree of cyanobacterial genomes and presence of key regulatory proteins</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-4.pdf">Fig. 4</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Alignment of HetR proteins</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-5.pdf">Fig. 5</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Amino acid sequences of candidate PatS proteins</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-6.pdf">Fig. 6</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Amino acid sequences of candidate PatX proteins</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-7.pdf">Fig. 7</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Conserved amino acid residues in PatS and PatX</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-8.pdf">Fig. 8</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Conserved nucleotides upstream from <I>patS</I>, <I>patX</I>, and DIF1-motif-containing regions</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-9.png" target="_blank">Fig. 9</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Kinetics of P<sub><I>patX</I></sub>-<I>gfp</I> and P<sub><I>patS</I></sub>-<I>gfp</I> reporters expression in <I>Anabaena</I> PCC 7120</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="PatX-MM-Fig-10.png" target="_blank">Fig. 10</a>: </span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
Ectopic overexpression of <I>patX</I> on diazotrophic growth and heterocyst 
differentiation in <I>Anabaena</I> PCC 7120
</span>
</p>
-->

<p class=MsoNormal style='margin-right:152.45pt;margin-bottom:
0in;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'><u>Supporting Figures</u> (<a href="FigS01-11.pdf">all in one file</a>)</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
Figures S01 - S08: Pathway maps, presence or absence of proteins of endosymbionts superimposed
<BR>Figure S01: Glycogen metabolism
<BR>Figure S02: Glycolysis
<BR>Figure S03: Pentose phosphate / Entner-Doudoroff pathways
<BR>Figure S04: Tricarboxylic acid cycle
<BR>Figure S05: Biotin biosynthesis
<BR>Figure S06: Folate and derivatives biosynthesis
<BR>Figure S07: Pantothenate /Coenzyme A biosynthesis
<BR>Figure S08: Pseudocobalamin biosynthesis
</span>
<span style='font-size:18.0pt;font-family:TimesNewRomanPS-BoldMT;mso-bidi-font-family:
TimesNewRomanPS-BoldMT;mso-bidi-font-weight:bold'>
</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
Figure S09: Alignment of small chlorophyll-&alpha; binding-like proteins </span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
Figure S10: Alignment of maltose transporter-like proteins</span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
Figure S11: &alpha;-glucosidase phylogeny</span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
</span>
</p>



<p class=MsoNormal style='margin-right:152.45pt;margin-bottom:
0in;margin-left:1.0in;margin-bottom:.0001pt;text-align:justify;mso-layout-grid-align:
none;text-autospace:none'>
<u><span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>Supporting Tables</span></u>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="Supplemental-Table-S1-genomes.xlsx">Table S1</a>: </span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
Genomes used in study</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="Supplemental-Table-S2-orthologs.xlsx">Table S2</a>: </span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
All orthologous proteins; also by selected metabolic pathways and biochemical capabilities</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="Supplemental-Table-S3-sets.xlsx">Table S3</a>:
<BR>Table S3A: Counts of sets of orthologs and their intersections
<BR>Table S3B: Proteins present in Core set with orthologs in no Rhopalodian Endosymbiont
<BR>Table S3C: Proteins with orthologs in all Rhopalodian endosymbionts but not in Rip8802
</span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="Supplemental-Table-S4-transport.xlsx">Table S4</a>: 
<BR>Table S4A: Proteins related to carbon transport and subsequent metabolism
<BR>Table S4B: Proteins related to nitrogen transport
</span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'>
</span>
</p>

<p style='margin-top:0in;margin-right:1.05in;margin-bottom:
0in;margin-left:135.0pt;margin-bottom:.0001pt;text-indent:-63.0pt;mso-layout-grid-align:
none;text-autospace:none'>
<span style='font-size:18.0pt;font-family:TimesNewRomanPSMT;
mso-bidi-font-family:TimesNewRomanPSMT'>
<span style='mso-spacerun:yes'>  </span>
<a href="Supplemental-Table-S5-proteins-for-tree.xlsx">Table S5</a>: Orthologs used to construct the genomic phylogenetic tree</span>
<span style='font-size:18.0pt;mso-bidi-font-weight:bold'></span>
</p>


</div>
</body>
</html>
